Source/Libs/GtkSharp/Bin.cs

```diff
@@ -24,6 +24,9 @@ namespace Gtk {
 	using System.Runtime.InteropServices;
 
 	public partial class Bin {
+
+		protected Bin() : base() { }
+
```

Comment:
why this change is needed?

the autogenerated code looked like this:

```
		protected Bin() : base(IntPtr.Zero)
		{
			CreateNativeObject (new string [0], new GLib.Value [0]);
		}
```

Reply:
Just to minimize memory allocations

Reply:
Reverted